modules/dnn/test/test_onnx_conformance.cpp

```diff
@@ -224,6 +224,7 @@ static const TestCase testConformanceConfig[] = {
     {"test_depthtospace_example", 1, 1},
     {"test_dequantizelinear", 3, 1},
     {"test_dequantizelinear_axis", 3, 1},
+    {"test_dequantizelinear_blocked", 1, 1},
```

Comment:
The official model has 3 inputs, so it should be `3`. Please do not modify the model and just take it from the official onnx repo.

---

You can modify onnx importer to take scale & zp as inputs if they cannot be found in constants. Then you need to modify the forward pass in the implemenetation as well.

Reply:
Done and update the PR description as well as the test cases!